### PR TITLE
merge Moebius axonal neuropathy hypogonadism

### LINKED ISSUE #7198: MONDO:0016819 & MONDO:0023820 Moebius axonal neuropathy hypogonadism[Merge]
**Mondo term (ID and Label)**
MONDO:0023820 Moebius axonal neuropathy hypogonadism 

**Reason for deprecation**
These appear to be the same concept, the MeSH record seems to point to the same concept as the Orphanet term based on the synonym/ definition for the MeSH entry.  https://www.ncbi.nlm.nih.gov/mesh/?term=C535806

**Term to be merged with**
MONDO:0016819 Moebius syndrome-axonal neuropathy-hypogonadotropic hypogonadism syndrome
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7198

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -366748,16 +366748,26 @@ relationship: has_characteristic MONDO:0021136 {source="MONDO:0015906"} ! rare
...
 subset: ordo_malformation_syndrome {source="Orphanet:2560"}
 subset: orphanet_rare {source="Orphanet:2560"}
 subset: rare
+synonym: "Moebius axonal neuropathy hypogonadism" EXACT [MONDO:0023820]
+synonym: "Moebius syndrome with hypogonadotrophic hypogonadism and progressive peripheral neuropathy axonal and demyelinating type" RELATED [GARD:0003698]
 xref: GARD:3698 {source="Orphanet:2560"}
+xref: MESH:C535806 {source="MONDO:equivalentTo"}
 xref: Orphanet:2560 {source="MONDO:equivalentTo"}
 xref: UMLS:C2931024 {source="Orphanet:2560/e", source="MONDO:equivalentTo", source="Orphanet:2560"}
+xref: UMLS:C2931024 {source="GARD:0003698", source="MONDO:relatedTo"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:366763]: These two xrefs to `UMLS:C2931024` should be combined.

**Review by twhetzel — COMMENTED** [review]
Do the xrefs with `UMLS:C2931024` need to be combined? See comment inline.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:366763]: yes, I merged these by running make norm. You'll see a few other edits on this PR b/c of the normalize. Thanks!

**Comment by nicolevasilevsky** [comment]
@twhetzel this is ready for re-review

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**